src/hid/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.1)
 include(../../cmake/bob_robotics.cmake)
-BoB_module(SOURCES joystick_linux.cc joystick_windows.cc
+BoB_module(SOURCES joystick_linux.cc joystick_windows.cc joystick_glfw_keyboard.cc
            BOB_MODULES common
            PLATFORMS linux windows)
```

Comment:
We could drop the ``PLATFORMS`` array now, because ``JoystickGLFWKeyboard`` should also work on Macs (I guess?)

Reply:
:+1: